shell/platform/windows/win32_dpi_helper.h

```diff
@@ -28,17 +28,35 @@ class Win32DpiHelper {
   // Wrapper for OS functionality to return the DPI for |HWND|
   UINT GetDpiForWindow(HWND);
 
+  // Wrapper for OS functionality to return the DPI for the System. Only used if
+  // Per Monitor V2 is not supported by the current Windows version.
+  UINT GetDpiForSystem();
+
   // Sets the current process to a specified DPI awareness context.
   BOOL SetProcessDpiAwarenessContext(DPI_AWARENESS_CONTEXT);
```

Comment:
Do we actually need three public functions here? Would it make sense to just expose a top-level `SetProcessDpiAware` (which would actually be what's currently called SetDpiAwarenessAllVersions, and the current function with that name being called something with `System` in it, maybe)? It's not clear to me if there are cases where a client would need to know the details of which mode is being set.

Reply:
Yeah I agree, we just need to expose the one that would set it for all versions.

Reply:
Done